[2024-02-05T07:22:05.708+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: sqlite_randomuser.creating_table manual__2024-02-05T07:22:04.711771+00:00 [queued]>
[2024-02-05T07:22:05.721+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: sqlite_randomuser.creating_table manual__2024-02-05T07:22:04.711771+00:00 [queued]>
[2024-02-05T07:22:05.721+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-02-05T07:22:05.722+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-02-05T07:22:05.722+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-02-05T07:22:05.737+0000] {taskinstance.py:1389} INFO - Executing <Task(SqliteOperator): creating_table> on 2024-02-05 07:22:04.711771+00:00
[2024-02-05T07:22:05.744+0000] {standard_task_runner.py:52} INFO - Started process 52505 to run task
[2024-02-05T07:22:05.746+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sqlite_randomuser', 'creating_table', 'manual__2024-02-05T07:22:04.711771+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/sqlite_randomuser.py', '--cfg-path', '/tmp/tmpe3pvmm38', '--error-file', '/tmp/tmpc2_ifyzl']
[2024-02-05T07:22:05.748+0000] {standard_task_runner.py:80} INFO - Job 17: Subtask creating_table
[2024-02-05T07:22:05.811+0000] {task_command.py:371} INFO - Running <TaskInstance: sqlite_randomuser.creating_table manual__2024-02-05T07:22:04.711771+00:00 [running]> on host fe2863c78e2f
[2024-02-05T07:22:05.897+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sqlite_randomuser
AIRFLOW_CTX_TASK_ID=creating_table
AIRFLOW_CTX_EXECUTION_DATE=2024-02-05T07:22:04.711771+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-05T07:22:04.711771+00:00
[2024-02-05T07:22:05.898+0000] {sqlite.py:59} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users(
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL PRIMARY KEY
            );
        
[2024-02-05T07:22:05.909+0000] {base.py:68} INFO - Using connection ID 'sql_lite' for task execution.
[2024-02-05T07:22:05.910+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS users(
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL PRIMARY KEY
            ), parameters: []
[2024-02-05T07:22:05.925+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=sqlite_randomuser, task_id=creating_table, execution_date=20240205T072204, start_date=20240205T072205, end_date=20240205T072205
[2024-02-05T07:22:05.963+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-05T07:22:06.009+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
